apps/cookbook/src/app/examples/link-example/examples/trailing-icon.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-link-example-trailing-icon',
+  template: `<a class="trailing-icon-example" target="_blank" href="https://github.com/kirbydesign/designsystem">Trailing icon</a>`,
+  style: `.trailing-icon-example
+{
+@include utils.trailing-icon("navigation")
+/* 
+@include utils.trailing-icon("https://cdn.jsdelivr.net/npm/open-iconic@1.1.1/svg/heart.svg")
+@include utils.trailing-icon()
```

Comment:
I don't think the parameterless example makes sense to the consumers of Kirby? Isn't this something we only use internally?
```suggestion
```

Reply:
You're right about that